docs/ARCHITECTURE.md

```diff
@@ -9,10 +22,50 @@ can be configured to run only during certain time periods or not at all. Setting
 make sure repair is spread out over the interval time while alarms are provided to signal when a job has not run for
 longer than expected.
 
+<div align="center">
```

Comment:
This chart looks really strange, see if you can get it more spaced out. Right now it looks like each ecchronos is connected to 2 cassandra nodes.